aiida_pseudo/cli/install.py

```diff
@@ -231,6 +231,23 @@ def cmd_install_pseudo_dojo(version, functional, relativistic, protocol, pseudo_
                 msg = f'md5 of pseudo for element {element} does not match that of the metadata {md5}'
                 echo.echo_critical(msg)
 
+        adjusted_cutoff_elements = []
```

Comment:
probably make this a dictionary with list for each stringency, such that in the warnings you can say explicitly for which stringency we had to make adjustments

Reply:
All elements that are missing cutoffs are missing them for all stringencies. So, this will make the output three times as verbose, but we can do it if you like.

Reply:
I realize that, but that is why I reformatted the warning messages to have a single message per stringency. This way there will be three lines with a warning at most, which I think is acceptable. This way we also are clear in the case in the future somehow the cutoffs are only missing for certain stringency sets